### ISSUE TITLE
CLI and HTML report not consistent

### ISSUE THREAD

**Issue by ribua** [RHsyseng/openshift-checks#131]
Hi , 

Reporting issues for checks that are working from CLI and report indicates the following messages. 

```bash
./openshift-checks.sh -s checks/zombies
Using system:serviceaccount:ribu-test:sa-kubeconfig context
Collecting zombie processes... (using oc debug, it can take a while)
No issues found
```

The above when run via HTML report just indicates the below information

![image](https://user-images.githubusercontent.com/41564422/218604531-a4bd57e3-8619-402e-84c6-83e91607b16a.png)

The page doesn;t refresh with the new information or final results. 

Ribu

**Comment by iranzo** [comment]
It reports the status as 'OK' in the HTML interface. The 'No issues found' is equivalent to the 'OK' as reported by the CLI interface:

~~~sh
  if [ ${errors} -gt 0 ]; then
    die "${RED}Total issues found: ${errors}${NOCOLOR}"
  else
    msg "${GREEN}No issues found${NOCOLOR}"
  fi
~~~

**Comment by iranzo** [comment]
Try and paste the output of `risu.py -l -i zombies -v` here

**Comment by gitkent** [comment]
Hi all, I am working with @ribua for this issue. Please find below output from `risu.py -l -i zombies -v`

```
❯ risu.py -l -i zombies -v
INFO:risu:Storing output on file reports/osc.json

[l%%%l]=[]=

# checksroot/nagios/checks/zombies: okay
```

**Comment by iranzo** [comment]
Hi @gitkent 

So that confirms what I was mentioning, the difference in the reporting comes from the openshift-checks.sh wrapper. Graphical or risu console output of 'okay'  is the equivalent of the wrapper saying 'No issues found'.

There was a PR for making the openshift-checks.sh more silent at #113
